frontend/src/components/ShootStatus.vue

```diff
@@ -137,22 +137,24 @@ export default {
       return this.operationType === 'Reconcile'
     },
     isUserError () {
-      return isUserError(this.errorCodes)
+      return isUserError(this.allErrorCodes)
     },
-    lastErrorDescription () {
-      return get(this.lastError, 'description')
-    },
-    errorCodes () {
-      return get(this.lastError, 'codes', [])
+    lastErrorDescriptions () {
+      return map(this.lastErrors, lastError => {
+        return {
+          description: lastError.description,
+          errorCodeDescriptions: map(lastError.codes, code => get(errorCodes, `${code}.description`, code))
+        }
+      })
```

Comment:
return map(this.lastErrors, lastError => ({
        description: lastError.description,
        errorCodeDescriptions: map(lastError.codes, code => get(errorCodes, `${code}.description`, code))
      }))